<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pApp</class>
 <widget class="QMainWindow" name="pApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>p</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="ButtonsLayout_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="ButtonsLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="PanelsLayout_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="PanelsLayout">
       <item>
        <widget class="QFrame" name="frame0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Score</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Score_Name"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Score_Duration">
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Parts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="PartsList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="bScoreLoad">
              <property name="text">
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bScoreSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Partiture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Part_Name"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Numerator</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Part_Numerator"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Denominator</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Part_Denominator"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Staves</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="StavesList"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Stave</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Stave_Name"/>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Build</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Stave_Build"/>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Condition</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Stave_Condition"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Notes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="NotesList"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Matrix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="StavesTable">
            <property name="styleSheet">
             <string notr="true">QHeaderView::section {
	/* Заголовки */
	background-color: rgb(230, 255, 255);
	text-align: center;
	padding: 0px;
	color: rgb(255, 0, 0);
}
QTableWidget::item {
	/* Ячейки */
	padding: 0px;
	color: rgb(0, 0, 255);
	text-align: center;
}
</string>
            </property>
            <row>
             <property name="text">
              <string>Str01</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>0</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>1</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>1212</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>matrix</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Note</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Column</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Note_Column"/>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Fxl, Pos, Pow</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="Note_Fxl">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Note_Pos">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Note_Pow">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="printdivider">
            <property name="text">
             <string>print divider</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
